nvml-wrapper/src/device.rs

```diff
@@ -289,6 +289,49 @@ impl<'nvml> Device<'nvml> {
         }
     }
 
+    /**
+    Gets the NUMA nodes physically close to the GPU.
+
+    Main goal is to facilitate memory placement optimisations for multi CPU/GPU settings.
+    Node (set) size needs to be something like `<Number of nodes> / (std::mem::size_of::<c_ulong>() / 8) + 1`
+
+    # Errors
+
+    * `Uninitialized`, if the library has not been successfully initialized
+    * `InvalidArg`, if this `Device` is invalid
+    * `NotSupported`, if this `Device` does not support this query
+    * `GpuLost`, if this `Device` has fallen off the bus or is otherwise inaccessible
+    * `Unknown`, on any unexpected error
+    */
+    // Checked against local
+    // Tested
+    #[cfg(target_os = "linux")]
+    #[doc(alias = "nvmlDeviceGetMemoryAffinity")]
+    pub fn memory_affinity(
+        &self,
+        size: usize,
+        scope: nvmlAffinityScope_t,
```

Comment:
Normally I'd say that we should be wrapping this in an enum, but I see that there are already other methods on this struct that directly take `nvmlAffinityScope_t` so I guess it's better to be consistent here.